scss/civicrm/dashboard/pages/_edit-dashboard.scss

```diff
@@ -0,0 +1,100 @@
+#help {
+  clear: both;
+  margin-top: 30px;
+  border: none;
+  box-shadow: $box-shadow;
+  background-color: $crm-help-success;
+  color: $crm-copy;
+}
+
+.dashlets-header {
+  font-size: $font-size-h3;
+  margin-bottom: 20px;
+}
+
+#available-dashlets {
+  width: auto;
+  border: 1px dashed $crm-dasboard-header;
+  background-color: $crm-white;
+  min-height: 40px;
+  padding: 25px;
+
+  .portlet {
+    border: none;
+
+    .portlet-header {
+      background: $crm-accordion-header-color;
+      margin: 0;
+      color: $crm-dark-blue;
+      padding: 15px;
+      font-size: $font-size-medium;
+      border-color: $crm-dark-grayblue;
+      font-family: $font-family-base;
+      font-weight: 600;
+      border-radius: 2px;
```

Comment:
Its also preferred to use a mixin to add browser prefixes

(somehting like this)
https://css-tricks.com/snippets/sass/mixin-prefix-properties/